### DISCUSSION TITLE
Announcing Umami Cloud BETA

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1672] [General]
Hi Everyone!

We are excited to announce that we are opening up our BETA program for the Umami Cloud. The Umami Cloud is a managed hosting service created by the developers of Umami. You can sign up to at https://cloud.umami.is. There is no cost to use it during the BETA period and we welcome your feedback!

## How does it work?

Simply go to https://cloud.umami.is/signup and create an account.

<img src="https://user-images.githubusercontent.com/477975/203228346-4c1e48fa-720b-498b-8df0-b50b68fd57cb.png" width="500" />

After verifying your email address, you can log in and create a website.

![image](https://user-images.githubusercontent.com/477975/203232057-230b94c1-8241-4ebd-8a71-dfdf6cbf1833.png)

Then under `Settings` you can get your tracking code and immediately start collecting stats for your website.

![image](https://user-images.githubusercontent.com/477975/203232792-593635e8-4542-4c2e-869b-e096341a1222.png)

Then click the `View` button to see your statistics.

![image](https://user-images.githubusercontent.com/477975/203233254-9fbf7a7e-80d3-4471-ba50-46700b2e2335.png)

## What is going to happen to Umami?

Nothing will change! We are developers dedicated to open-source and Umami will always be free and open. Whatever features we add to Umami Cloud will be open-source and committed back to the project.

## What will the pricing be?

We are still working on this, but we expect to have affordable pricing tiers. And of course, self-hosting will always be free.

## How can I help?

Try out Umami Cloud and let us know what you think. You are free to use the discussions and issues here, join our Discord channel (https://discord.gg/4dz4zcXYrQ) or email us directly at support@umami.is.

**Comment by devarshishimpi** [comment]
@mikecao Had a question here,
If ad blockers like ( U-block origin ) block umami cloud's URL, then even umami won't be able to track all page views, how is this going to be worked?

**Reply by joeyyax** [reply]
Nothing's foolproof, but support for CNAME forwarding would be neat.

**Reply by mikecao** [reply]
You can technically host the tracker script anywhere, even on your own site.

**Reply by devarshishimpi** [reply]
True

**Comment by joeyyax** [comment]
Is/Will there be support for migrating data to/from existing Umami setups?

**Reply by mikecao** [reply]
Yes, we are working on import tools so you can migrate data into cloud.

**Comment by TigersWay** [comment]
Needed is a way to export a journal/log to be able to catch and I hope understand the discrepancy between umami and others!
